src/actions/authActions.ts

```diff
@@ -22,22 +25,20 @@ export const signUp = (data: SignUpData, onError: () => void): ThunkAction<void,
       const res = await fireAuth.createUserWithEmailAndPassword(data.email, data.password);
       if (res.user) {
         console.log('created new user');
-        const userData: User = {
-          email: data.email,
-          firstName: data.firstName,
-          id: res.user.uid,
-          createdAt: Date.now(),
-        };
+        // const userData: User = {
+        //   email: data.email,
+        //   firstName: data.firstName,
+        //   id: res.user.uid,
+        //   createdAt: Date.now(),
+        // };
         // await firebase.firestore().collection('/users').doc(res.user.uid).set(userData);
         // await res.user.sendEmailVerification();
         // dispatch({
         //   type: NEED_VERIFICATION,
         // });
         // TODO: SEND THIS DATA TO THE DATABASE
-        dispatch({
-          type: SET_USER,
-          payload: userData,
-        });
+
+        getHistory().push(AppPaths.SIGN_IN);
       }
     } catch (err) {
       console.log(err);
```

Comment:
a todo we should add is coming up with a better way to gracefully handle errors. I like the idea of having a banner element that reports back success and failure to the user.

Reply:
This is a little unrelated to this PR. We do show the user the error we get. It just shows us as plain text on the page and it is not styled using a banner.

Reply:
Banner to be implemented in a separate PR
Relates issues: #18 #11